### ISSUE TITLE
Formatter Error - No such formatter found:  email

### ISSUE THREAD

**Issue by RudyOnRails** [olifolkerd/tabulator#1641]
**Describe the bug**
When I use the `email` formatter, I get an error in the console:
```
tabulator.js:14282 Formatter Error - No such formatter found:  email
```

**Tabulator Info**
- Which version of Tabulator are you using?
`"tabulator-tables": "^4.1.3"`
- Post a copy of your construct object if possible so we can see how your table is setup
```
        @tabulator = new Tabulator(@$refs.table, {
          columns: [
            title:'Full Name'
            field:'full_name'
          ,
            title:'Email'
            field:'email'
            formatter:'email'
          ]
          data: data.data.users
          layout:'fitColumns'
        })
```

**To Reproduce**
1. setup a basic table that includes email addresses
2. add the email formatter
3. open console and see error

**Expected behavior**
I expected it to render data as an anchor with a mailto: link to the given value

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS 10.13.6
 - Browser chrome
 - Version 70.0.3538.110 (Official Build) (64-bit)

**Additional context**

**Comment by puffsnow** [comment]
Hi @RudyOnRails,

I think it is not a bug. There is not a 'email' formatter described in the document, but you can achieve this goal in another way:
In the http://tabulator.info/docs/4.0/format#format-builtin page, `link - urlPrefix` mentioned that ` to turn a emaill address into a clickable mailto link you should set this to "mailto:"`. So maybe you can try this:
```
@tabulator = new Tabulator(@$refs.table, {
          columns: [{
            title:'Full Name',
            field:'full_name'
          }, {
            title:'Email',
            field:'email',
            formatter:'link',
            formatterParams: {urlPrefix:"mailto:"}
          }],
          data: data.data.users,
          layout:'fitColumns'
        })
```
I hope this solution can help you. : )

**Comment by RudyOnRails** [comment]
Hi @puffsnow,

Thank you for the idea. However, am I missing something? I see `email` listed as a formatter at: http://tabulator.info/examples/4.1#formatters - that's why I filed a bug. What are your thoughts after viewing that link?

Thanks,
Kevin

**Comment by puffsnow** [comment]
Hi @RudyOnRails,

I see, and yes, I believe there are some mistakes because I didn't see email formatter in source code. (even 4.1.3). The code or document has to be fixed.

**Comment by olifolkerd** [comment]
Hey @RudyOnRails 

That was a documentation issue, the ***email*** formatter was merged into the ***link*** formatter in v4.0

The main documentation was updated, but i must have missed that bullet point in the example.

Cheers

Oli :)
